src/settings/MappingProfiles/MappingProfilesTransformationsModal/TransformationsField/TransformationFieldGroup/TransformationFieldGroup.js

```diff
@@ -79,6 +95,10 @@ export const TransformationFieldGroup = ({
             aria-invalid={!isValid}
             marginBottom0
             onChange={event => {
```

Comment:
Move onChange functionality out of render

Reply:
> Move onChange functionality out of render

done!